### PR TITLE
Update velero image & boilerplate

### PR DESCRIPTION
**Pull request by wanghaoran1988**
This includes two changes:
1. Update the boilerplate
2. Update the velero image to 1.7.1
3. Use micro ubi image
4. update go dependencies.

### CODE REVIEW

**Comment by cblecker** [comment]
Do we need to update the golang imports?

File: build/Dockerfile [hunk 1]
```diff
@@ -8,7 +8,7 @@ COPY . .
 RUN make go-build
 
 ####
-FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+FROM registry.access.redhat.com/ubi8/ubi-micro:latest
```
Comment (dustman9000) [inline build/Dockerfile:11]: What's the reason for this change?

Comment (wanghaoran1988) [reply build/Dockerfile:11]: From the doc here: https://www.redhat.com/en/blog/introduction-ubi-micro
It has smaller image size, and can reduce the attack surface.

**Comment by cblecker** [comment]
/lgtm
/approve
/hold

Holding if @dustman9000 has any thoughts/feedback.
